extensions/mongodb/pom.xml

```diff
@@ -124,6 +123,12 @@
             <artifactId>calcite-core</artifactId>
             <version>${calcite.version}</version>
             <scope>provided</scope>
+            <exclusions>
```

Comment:
Why this exclusion?

Reply:
ups, leftover. 0cbc67e79fcd57a52c3b880b8c2c0513eb75dc42